packages/components/menu/src/menu.ts

```diff
@@ -286,6 +289,10 @@ export default defineComponent({
       initMenu()
       if (props.mode === 'horizontal') {
         new Menubar(instance.vnode.el!, nsMenu.namespace.value)
+
+        if (props.ellipsis) {
```

Comment:
Please move it to `watch`, since it should be updated when `props.ellipsis` changes.